migrations/2023-10-02-145002_community_followers_count_federated/up.sql

```diff
@@ -0,0 +1,34 @@
+-- The subscriber count should only be updated for local communities. For remote
+-- communities it is read over federation from the origin instance.
+CREATE OR REPLACE FUNCTION community_aggregates_subscriber_count ()
```

Comment:
Looks good. Doesn't look like we can use the trigger, since `community_follower` doesn't have a local column on it.